src/rules/style.ts

```diff
@@ -59,6 +67,8 @@ export default {
   'prefer-function-over-method': true,
   // Prefer foo(): void over foo: () => void in interfaces and types.
   'prefer-method-signature': true,
+  // Prefer a switch statement to an if statement with simple === comparisons.
+  'prefer-switch': false,
```

Comment:
👍 to `false`.

This is bad guidance.  Convert that 💩 into a well-named `const`.

[Examples](https://github.com/palantir/tslint/blob/master/test/rules/prefer-switch/default/test.ts.lint).